datadog-setup.php

```diff
@@ -1099,12 +1099,16 @@ function check_library_prerequisite_or_exit($requiredLibrary)
  * @param string $extName E.g. json
  * @return void
  */
-function check_php_ext_prerequisite_or_exit($binary, $extName)
+function check_php_ext_prerequisite($binary, $extName)
 {
     $extensions = shell_exec(escapeshellarg($binary) . " -m");
 
+    // See: https://github.com/DataDog/dd-trace-php/issues/2787
+    if ($extensions == null || strpos($extensions, '[PHP Modules]') === false) {
+        echo "WARNING: The output of '$binary -m' could not be reliably checked. Please make sure you have the PHP extension '$extName' installed.\n";
+    }
     if (!in_array($extName, array_map("trim", explode("\n", $extensions)))) {
-        print_error_and_exit("Required PHP extension '$extName' not found.\n");
```

Comment:
I think it's still fine to fail _if_ the command actually goes through?

Reply:
Fair point

Reply:
done